spatial-history/index.md

```diff
@@ -1,39 +1,33 @@
 
 
 <div class="markers">
   <!-- these buttons hide/show all the markers  -->
   <!-- to hide/show blue or red markers instead, change my_markers below to blue_markers
        to red_markers.  If you have defined your own color (or other) arrays, use those instead -->
-  <button onclick="toggleMarkers(my_markers, my_map)" class="rounded" id="hide">Toggle markers</button>
-  <button onclick="showMarkers(my_markers, my_map)" id="show"> show markers</button>
+
 </div>
   <div id="mapcontainer">
     <div id="map_canvas"></div>
   </div>
   <div id="map_legend"></div>
 </div>
 
-# Next Section
+<h2 id=history>History</h2>
+Looking at the creation of the Montreal video game cluster, it is clear that it began with the attraction of French gaming company Ubisoft to the city in 1997, as it would become one of the largest game publishers worldwide and it was after this that the influx of other large developers occurred. The draw for Ubisoft to Montreal was a series of policies financially incentivizing multimedia development in the city, particularly for French-language companies ([Pilon, pg 4](https://www.hindawi.com/journals/usr/2013/957630/)). Building off the success of Ubisoft Montreal and the creation of a number of quality assurance and other auxiliary companies, the city provided greater subsidies and professional training programs which, combined with the growing pool of skilled, creative game developers, established the city as a space for creators to find employment and for studios to have access to a larger selection of talented employees ([Grandadam, pg 1706](https://journals-scholarsportalinfo.myaccess.library.utoronto.ca/pdf/00343404/v47i0010/1701_psatdotvgiim.xml/)). These advantages caused the industry in Montreal to grow, but this growth became exponential as the city became enriched with third party companies able to provide necessary services for developers, allowing the entire development process to be carried out locally ([Darchen, pg 317](https://journals-scholarsportal-info.myaccess.library.utoronto.ca/pdf/09654313/v23i0002/311_pfccacgiimam.xml)). This produced the effect we can see on the map, which is a number of large-scale video game development clustered in a small radius, forming a cultural sphere of creative development intrinsically linked to the space it occupies.
```

Comment:
:heavy_check_mark: some word choices I would change: "Looking at hte creation of..." is a non-=specific clause; replace "exponential growth" w/ "positive feedback loop" or similar language.  Nice call-out to the map.